components/AddVehicleModal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+
+class AddVehicleModal extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            carName: '',
+            arduinoID: ''
```

Comment:
i thought arduino id was gonna be a number, not a string...
if so, let's make this default value like null or something and then store it as a number type